server/routes/aboutPop/locale.json

```diff
@@ -17,6 +25,64 @@
     "subNav": {
       "about": "About",
       "summary": "Plan summary"
-    }
+    },
+    "areas": [
+      {
+        "area": "Thinking, behaviours and attitudes",
+        "crimNeedsKey": "thinkingBehaviourAndAttitudes",
+        "crimNeedsSubKey": "think",
+        "assessmentKey": "thinking_behaviours_attitudes",
+        "goalRoute": "thinking-behaviours-and-attitudes"
+      },
+      {
+        "area": "Alcohol use",
+        "crimNeedsKey": "alcoholMisuse",
+        "crimNeedsSubKey": "alcohol",
+        "assessmentKey": "alcohol_use",
+        "goalRoute": "alcohol-use"
+      },
+      {
+        "area": "Health and wellbeing",
+        "crimNeedsKey": "healthAndWellbeing",
+        "crimNeedsSubKey": "emu",
+        "assessmentKey": "health_wellbeing",
+        "goalRoute": "health-and-wellbeing"
+      },
+      {
+        "area": "Accommodation",
+        "crimNeedsKey": "accommodation",
+        "crimNeedsSubKey": "acc",
+        "assessmentKey": "accommodation",
+        "goalRoute": "accommodation"
+      },
+      {
+        "area": "Education and employment",
+        "crimNeedsKey": "educationTrainingEmployability",
+        "crimNeedsSubKey": "ete",
+        "assessmentKey": "employment_education",
+        "goalRoute": "employment-and-education"
+      },
+      {
+        "area": "Finance",
```

Comment:
We've changes this to `Finances` as part of the design.

Reply:
Changed